examples/7guis/tempconv.slint

```diff
@@ -14,7 +14,10 @@ TempConv := Window {
                 }
             }
         }
-        Text { text: "Celcius = "; }
+        Text {
+            text: "°Celcius = ";
```

Comment:
Just a note that the screenshot at https://eugenkiss.github.io/7guis/tasks#temp doesn't have `°`